ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -31,6 +31,9 @@
       .put("org-mozilla-vrbrowser-wavevr", "1614411") //
       .build();
 
+  private static final Set<String> FIREFOX_ONLY_DOCTYPES = new HashSet<>(
+      Arrays.asList("event", "main", "modules"));
```

Comment:
I'd prefer a Guava ImmutableSet, which would prevent accidental changes and can be constructed a bit more elegantly.